### DISCUSSION TITLE
About the jumper settings

### DISCUSSION THREAD

**Discussion by Roland-swe** [ctag-fh-kiel/ctag-tbd#67] [Q&A] [answered]
Hello. I stumbled across this project recently and I have a question about the jumper settings for signal routing of the inputs and outputs. I have not yet found any description on those, but saw in another post that they should be 2-3 in all cases. 
Are the OUT3, OUT4 and the AUXR and AUXL not used at all at the moment?

Since I am a banana person I altered the footprints for the jack so they can be used with either 3.5mm jacks or banana on the units I will order. I am glad there were no routing on the back side of the jacks :-)

**Comment by ctag-fh-kiel** [comment, marked as answer]
Hi there, correct OUT3,4 & AUXR,L are unused, you could attach wires to them, alter the driver and start using them :)
Driver code is located here https://github.com/ctag-fh-kiel/ctag-tbd/blob/master/components/drivers/codec.cpp
Let us know how happy you are with your modded banana version.

**Comment by Roland-swe** [comment]
Thanx! Will do.
